lib/trello/board.rb

```diff
@@ -154,13 +154,11 @@ def remove_member(member)
     # Returns a reference to the organization this board belongs to.
     one :organization, path: :organizations, using: :organization_id
 
-    def labels names=true
-      if names
-        labels = client.get("/boards/#{id}/labelnames").json_into(LabelName)
-      else
-        labels = client.get("/boards/#{id}/labels").json_into(Label)
-      end
-      MultiAssociation.new(self, labels).proxy
+    many :labels
+
+    def label_names
```

Comment:
I left the old call because it was still a valid API call and yielded a different result on older boards. But I have to recheck. Its over six month already.